pages/platform/databases/postgresql_tuto_01_connect_strapi_to_managed_postgresql/guide.en-asia.md

```diff
@@ -274,8 +276,7 @@ You can start by doing:
 Done in 662.54s.
 </code></pre>
 
-You could now start you strapi application with yarn commands as described above, but before that, we need to modify an SSL parameter.<br>
-If you don't, this is what will happen:
+If you prefered choose `No` when prompted for enabling SSL connection, this is what happen:
```

Comment:
If you prefer to choose `No` when prompted to enable SSL connection, this is what happens:

Reply:
Thx